fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -935,6 +948,7 @@ void DLApi::init() {
 	                   fdbCPath,
 	                   "fdb_tenant_verify_blob_range",
 	                   headerVersion >= ApiVersion::withTenantBlobRangeApi().version());
+	loadClientFunction(&api->tenantGetId, lib, fdbCPath, "fdb_tenant_get_id", headerVersion >= 720);
```

Comment:
I think this is going to need to be 730, but I'm also wondering if this whole function should be using the version aliases rather than specifying the versions directly. Maybe @sfc-gh-dzhou would know.

```suggestion
	loadClientFunction(&api->tenantGetId, lib, fdbCPath, "fdb_tenant_get_id", headerVersion >= 730);
```